Google.PowerShell/Google.PowerShell.csproj

```diff
@@ -23,6 +23,7 @@
     <ErrorReport>prompt</ErrorReport>
     <WarningLevel>4</WarningLevel>
     <DocumentationFile>bin\Debug\Google.PowerShell.XML</DocumentationFile>
+    <!--Don't create warnings for uncommented public members.-->
```

Comment:
Nit: I'd put space after `<--` and before `-->` for readability. Also, including the exact warning text is useful as well. But ultimately it doesn't matter.